Modules/Filtering/DisplacementField/include/itkDisplacementFieldJacobianDeterminantFilter.h

```diff
@@ -67,10 +67,10 @@ namespace itk
  * itk::Image<TRealType, N>.
  *
  * \par Filter Parameters
- * The method SetUseImageSpacingOn will cause derivatives in the image to be
+ * The method UseImageSpacingOn will cause derivatives in the image to be
```

Comment:
Since this commit is mostly dealing with `ImageSpacing` ivars, updating commit message to mention that would be helpful if anyone was searching for this change later. Perhaps:
```
ENH: Use `itkBooleanMacro` for `ImageSpacing` ivar in many classes

Use `itkBooleanMacro` to set boolean member variable values to On/Off:
- Deprecate `Set*(On/Off)` signatures.
- Test the methods.
```

Reply:
Looks reasonable. Changed in 7961565.